docs/src/__examples__/Tooltip/INDICATION.tsx

```diff
@@ -25,8 +25,10 @@ export default {
                 <Text>An open source design system for your next travel project.</Text>
               </Stack>
             }
+            aria-label="Orbit design system"
+            aria-labelledby="orbit-success"
           >
-            <Text>Orbit</Text>
+            <Text id="orbit-app">Orbit</Text>
```

Comment:
I am not sure about this whole file, in case it's necessary, I can revert it back.